src/foam/nanos/dig/SUGAR.js

```diff
@@ -279,12 +242,12 @@ foam.CLASS({
       name: 'postButton',
       label: 'Send POST Request',
       code: async function() {
-        if ( this.postURL !== '' ) {
+        if ( this.sugarURL !== '') {
           var req = this.HTTPRequest.create({
-            url: window.location.protocol + '//' + window.location.hostname + ':' + window.location.port + this.postURL + this.postData+ "&sessionId=" + localStorage.defaultSession,
+            url: window.location.protocol + '//' + window.location.hostname + ':' + window.location.port + this.sugarURL + '?data=' + this.postData+ '&sessionId=' + localStorage.defaultSession,
```

Comment:
postData should not be included twice in this .. it should be sufficient to include the postData in the payload and not in the URL, which would actually make this unusable if actual JSON was included in the postData since you are not encoding it.